src/styles/blocks/card.scss

```diff
@@ -0,0 +1,105 @@
+.card {
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  transition: transform 300ms;
+
+  &__image {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    color: $main-color;
+    font-size: 12px;
+    font-weight: 500;
+    font-style: normal;
+    margin-bottom: 4px;
+    line-height: 18px;
+  }
+
+  &__code {
+    display: flex;
+    color: $secondary-color;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__block-stars {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    color: $main-color;
+    text-align: right;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-text {
+      color: $secondary-color;
+      font-size: 12px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 18px;
+    }
+
+    &-sum {
+      color: $main-color;
+      font-size: 16px;
+      font-style: normal;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &:hover {
+    transform: scale(1.2);
+
+    & .card__name {
+      color: #34568b;
+    }
+  }
+
+  &__buy-link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+    background-color: $link-color;
+    color: $white-color;
+    border-radius: 5px;
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    text-decoration: none;
+    border: 1px solid $link-color;
+    box-sizing: border-box;
```

Comment:
Not fixed.
![image](https://github.com/mate-academy/layout_catalog/assets/102293862/49d89c5b-6f3a-47c1-adbb-7d659e792592)

You can set it in normalize.scss for all the elements using the "*" selector:
`* { box-sizing: border-box; }`